suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectCoins } from '@suite-common/wallet-core';
+import { useFormatters } from '@suite-common/formatters';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { Card, Table, Td, Text, Th, Tr, VStack } from '@suite-native/atoms';
+import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
+import { FiatCurrency } from '@suite-common/suite-config';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+
+type TransactionDetailValuesSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    fiatCurrency: FiatCurrency;
+    onSheetVisibilityChange: () => void;
+};
+
+/**
+ * Creates table header cell which contains percentual difference of transaction profit/loss.
+ */
+const renderTodayHeaderCell = (historicalPrice: string | number, actualPrice: string | number) => {
+    const fiatTotalHistoryNumeric = Number(historicalPrice);
+    const fiatTotalActualNumeric = Number(actualPrice);
+
+    const hasPriceIncreased = fiatTotalHistoryNumeric < fiatTotalActualNumeric;
+    const renderPercentage = fiatTotalHistoryNumeric !== fiatTotalActualNumeric;
+    let percentageDifference;
+    if (hasPriceIncreased) {
+        percentageDifference = Math.round(
+            (fiatTotalHistoryNumeric / 100) * (fiatTotalActualNumeric - fiatTotalHistoryNumeric),
+        );
+    } else {
+        percentageDifference = Math.round(
+            (fiatTotalActualNumeric / 100) * (fiatTotalHistoryNumeric - fiatTotalActualNumeric),
+        );
+    }
+
+    return (
+        <Text variant="hint" color="gray600">
+            Today{' '}
+            {renderPercentage && (
+                <Text variant="hint" color={hasPriceIncreased ? 'green' : 'red'}>
+                    {`${hasPriceIncreased ? '+' : '-'}${percentageDifference}`}%
+                </Text>
+            )}
+        </Text>
+    );
+};
+
+export const TransactionDetailValuesSheet = ({
+    isVisible,
+    onSheetVisibilityChange,
+    transaction,
+    fiatCurrency,
+}: TransactionDetailValuesSheetProps) => {
+    const { FiatAmountFormatter } = useFormatters();
+    const coins = useSelector(selectCoins);
+
+    const transactionInput = formatNetworkAmount(transaction.amount, transaction.symbol);
```

Comment:
You should not need `formatNetworkAmount` at all. Please use formatter instead.

Also get rid of these variables and use formatters directly in render:

```tsx
<CryptoAmountFormatter value={transaction.amount} symbol={transaction.symbol} />
```

Same with all others variables here.

Reply:
commented as TODO in 595577f78

Reply:
commented as TODO in https://github.com/trezor/trezor-suite/commit/595577f78b4355315f8b8ab716104ed58e366b10